personal_assistant/contacts/templates/contacts/add_contact.html

```diff
@@ -0,0 +1,44 @@
+{% extends "pa_core/home.html" %} {% block title%}
+    Create contact
+{% endblock %}
+
+{% block content %}
+
+<form method="POST" action="{% url 'contacts:add_contact' %}">
+    {% csrf_token %}
+    <div style="padding: 10px">
+        <label> Contact name:
+            {{ form.name }}
+        </label>
+        <span>{{ form.errors.name }}</span>
+    </div>
+    <div style="padding: 10px">
+        <label> Contact address:
```

Comment:
Так як у контексті сторінки зрозуміло до якої сутності відносяться всі поля то можна скоротити підписи до всіх полів видаливши перше слово з них:
```suggestion
        <label>Address:
```